util/minetestmapper.py

```diff
@@ -202,8 +271,30 @@ def usage():
 
 # Load color information for the blocks.
 colors = {}
+colors_path = "colors.txt"
+
+os_name="linux"
+if (os.path.sep!="/"):
+    os_name="windows"
+
+profile_path = None
+if os_name=="windows":
+    profile_path = os.environ['USERPROFILE']
```

Comment:
?? I'm pretty sure this isn't necessary, Windows has %HOME% as an alias for %USERPROFILE%